Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackFindingAlgorithm.hpp

```diff
@@ -90,8 +92,13 @@ class TrackFindingAlgorithm final : public BareAlgorithm {
   void computeSharedHits(const source_link_accessor_container_t& sourcelinks,
                          TrackFinderResult&) const;
 
+  ActsExamples::ProcessCode finalize() const override;
+
  private:
   Config m_cfg;
+
+  mutable std::atomic<size_t> m_nTotalSeeds;
+  mutable std::atomic<size_t> m_nFailedSeeds;
```

Comment:
Before I merge this, let me see if @HadrienG2 kills me over this. 😅

Reply:
Nah, that makes total sense :)

If you really want me to nitpick, you don't really need seq_cst semantics here, so fetch_add(1, std::memory_order_relaxed) would be a more efficient alternative to ++. But I don't think the code path you're running those increments is anywhere near hot enough to justify this sort of fine tuning.